### PR TITLE
refactor(directory): update console directory for readability

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA 
+ ) Connect more link url to anchor of CollectorPluginContents component

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/modules/CollectorPluginContents.vue [hunk 1]
```diff
@@ -53,6 +56,7 @@ const state = reactive({
     description: computed(() => props.plugin?.tags.description ?? ''),
     labels: computed<string[]>(() => props.plugin?.labels ?? []),
     isBeta: computed(() => !!props.plugin?.tags.beta ?? false),
+    pluginDetailLink: computed<CollectorPluginModel['tags']['link']>(() => props.plugin?.tags?.pluginDetailLink ?? ''),
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/modules/CollectorPluginContents.vue:59]: issue: I think the data of `pluginDetailLink` is from `tags.link`.

**Review by WANZARGEN — COMMENTED** [review]
Thanks for updating it. I left a small comment so please check it. :)

**Review by yuda110 — APPROVED** [review]
LGTM!!! Thank you for changing directories

**Review by WANZARGEN — APPROVED** [review]
Cool! :) I left a small suggestion!

**Pull request merged**